.secrets.baseline

```diff
@@ -162,11 +162,7 @@
         "filename": "apps/bbsync/tests/test_integration.py",
         "hashed_secret": "3c3b274d119ff5a5ec6c1e215c1cb794d9973ac1",
         "is_verified": false,
-<<<<<<< HEAD
-        "line_number": 70,
-=======
-        "line_number": 80,
->>>>>>> 7943700 (keep generating BZ ID Jira tracker label)
```

Comment:
nitpick: I believe this change belongs to the last commit.

Reply:
Oh :smile: I had to screw the rebasing